lib/pages/chat/events/message_content.dart

```diff
@@ -155,34 +155,26 @@ class MessageContent extends StatelessWidget {
             }
             return MessageDownloadContent(
               event,
-              textColor,
-              controller: controller,
+              onFileTapped: ({required Event event}) {
```

Comment:
change the controller function to same with onFileTapped, so just pass the function, no need call it
```suggestion
              onFileTapped: controller.onFileTapped
```